RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelDigisClustersFromSoA.cc

```diff
@@ -33,17 +33,25 @@ class SiPixelDigisClustersFromSoA : public edm::global::EDProducer<> {
 
   edm::EDPutTokenT<edm::DetSetVector<PixelDigi>> digiPutToken_;
   edm::EDPutTokenT<SiPixelClusterCollectionNew> clusterPutToken_;
+
+  //! Clustering-related quantities:
+  const int theClusterThreshold;     // Cluster threshold in electrons
+  const int theClusterThreshold_L1;  // Cluster threshold in electrons for Layer 1
```

Comment:
It's great that you bring this up, as I wanted to ask about this earlier. I thought that since the SoA version runs on the CPU, it should not use / know about data structures, constants defined for the GPU algorithms.
Parameters that are somehow connected are usually not grouped together (e.g. in helper structs) in CPU classes, so I thought that should stay like that.
However since the GPU version has to propagate these constants, it was a better choice, to use a helper struct.